### ISSUE TITLE
PHP Notice – yii\base\ErrorException Trying to get property 'uid' of non-object when creating or editing stencil

### ISSUE THREAD

**Issue by Antimated** [verbb/formie#239]
**Description**
Whenever I try to create or edit a stencil I get the following error:
![1606469219454](https://user-images.githubusercontent.com/2869409/100433551-86872500-309b-11eb-81a7-5ee87a951da9.png)

**Steps to reproduce**
1. Create or edit a stencil

### Form settings
Please pick from the options below.

- Multi-page form: n/a
- Submission Method: n/a
- Client-side Validation: n/a
- Custom Form Templates: no

**Additional info**
- Plugin version: 1.3.1
- Craft version: Craft Pro 3.5.15.1
- Multi-site: Yes

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by Antimated** [comment]
When is this update planned to be released? Not sure if I want to use the dev build :)

**Comment by engram-design** [comment]
Probably in 24 hours, I'll reply here when released.

**Comment by engram-design** [comment]
Fixed in [1.3.2](https://github.com/verbb/formie/releases/tag/1.3.2)
